charts/backingservices/charts/constellation-messaging/values.yaml

```diff
@@ -27,12 +36,36 @@ securityContext:
   readOnlyRootFilesystem: false
   # set allowPrivilegeEscalation to false to Restrict container from acquiring additional privileges https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
   allowPrivilegeEscalation: true # false
-serviceType: NodePort
+
+# Service
+service:
+  port: 3000
+  targetPort: 3000
+  serviceType: NodePort
 
 # An ingress will be provisioned if a hostname is defined, or omitted if the hostname is empty.
 # ingressClassName and annotations are optional and will be included if defined.
 # Due to the diverse requirements for ingresses and TLS configuration, it may be necessary to define the ingress separately from this chart.
 ingress:
-  domain:
```

Comment:
@alemax22 , Could you add "enabled" to true similar to constellation ? as used in https://github.com/pegasystems/pega-helm-charts/pull/770/files#diff-a986186add6bda5f00bf46ded3f50ba492c52674874c7e608ee0744c30f081d0R4